src/css/component.matrix.css

```diff
@@ -0,0 +1,118 @@
+/* ---------- COMPONENT matrix ---------- */
+
+.matrix {
+  --matrix-gap-x: var(--size-4);
+  --matrix-gutters-x: var(--size-3);
+  --matrix-item-size: var(--size-72);
+
+  @media (--mq-sm) {
+    --matrix-gap-x: var(--size-7);
+    --matrix-gutters-x: var(--size-5);
+    --matrix-item-size: var(--size-80);
+  }
+
+  @media (--mq-md) {
+    --matrix-gutters-x: var(--size-7);
+    --matrix-item-size: var(--size-96);
+  }
+
+  @media (--mq-lg) {
+    --matrix-item-size: var(--size-72);
+  }
+
+  @media (--mq-xl) {
+    --matrix-item-size: var(--size-80);
+  }
+}
+
+.matrix__list {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.matrix__link {
+  color: var(--color-white);
+  text-decoration: none;
+  transition: color var(--animation-duration) var(--animation-easing);
+}
+
+.matrix__link:hover,
+.matrix__link:active {
+  text-decoration: underline;
+}
+
+@media (--mq-to-lg) {
+  .matrix {
+    display: flex;
+    margin-left: calc(-1 * var(--matrix-gutters-x));
+    margin-right: calc(-1 * var(--matrix-gutters-x));
+    overflow-x: auto;
+    scroll-snap-type: x mandatory;
+  }
+
+  .matrix__list {
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    padding: 0 var(--matrix-gutters-x);
+  }
+
+  .matrix__list > li {
+    display: flex;
+    flex-direction: column;
+    margin-right: var(--matrix-gap-x);
+    min-height: var(--matrix-item-size);
+    scroll-snap-align: center;
+    width: var(--matrix-item-size);
+  }
+
+  .matrix__list > li:last-child {
+    margin-right: 0;
+  }
+}
+
+@media (--mq-lg) {
+  .matrix__list {
+    display: grid;
+    grid-template-columns: repeat(4, var(--size-1-4));
+  }
+
+  .matrix__list > li {
+    min-height: var(--matrix-item-size);
+  }
+
+  .matrix__list > li:nth-last-child(4):first-child {
+    grid-column: 1 / span 2;
+    grid-row: 1 / span 2;
+  }
+
+  .matrix__list > li:nth-last-child(4):first-child ~ :nth-child(2) {
+    grid-column: 3 / span 2;
+  }
+
+  .matrix__list > li:nth-last-child(4):first-child ~ :nth-child(3) {
```

Comment:
Just a note: Might be one to document to ensure that content editors understand the variants that exist, and how it behaves with differening amounts of items. (Are we shooting to support variations with just 3 and 4 items for now?)

Reply:
Yeah, I've based this on the designs. I only saw a 3 and 4 layout.

It's a relatively easy fix to add CSS for more or less.

I can add comments to the README's for pages that use matrix modules defining the use of 3 or 4 modules.